docroot/modules/custom/mass_admin_pages/src/Controller/AuthorHomePageController.php

```diff
@@ -16,6 +16,15 @@ class AuthorHomePageController extends ControllerBase {
    *   A simple renderable array.
    */
   public function authorHome() {
+    $user_data = \Drupal::service('user.data');
+    if (empty($user_data->get('tfa', \Drupal::currentUser()->id(), 'tfa_totp_seed'))) {
```

Comment:
Is there not an API function for getting this info? Getting it from user_data is very low level. Its rarely the right way.

Reply:
This is the official documentation for that https://www.drupal.org/node/1852360